package.json

```diff
@@ -36,21 +30,21 @@
     "validate": "npm run lint && npm test && npm run e2e && git status"
   },
   "dependencies": {
-    "which": "2.0.2"
+    "which": "4.0.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.23.7",
-    "@babel/eslint-parser": "^7.23.3",
+    "@babel/core": "^7.24.7",
+    "@babel/eslint-parser": "^7.24.3",
     "eslint": "^8.56.0",
     "eslint-config-tjw-base": "^2.0.0",
     "eslint-config-tjw-jest": "^2.0.0",
     "eslint-config-tjw-jsdoc": "^1.0.5",
-    "eslint-plugin-jsdoc": "^48.0.2",
+    "eslint-plugin-jsdoc": "^48.2.13",
     "fs-extra": "8.1.0",
     "get-windows-shortcut-properties": "^1.3.0",
-    "jest": "24.9.0",
+    "jest": "29.7.0",
```

Comment:
Jest > 24.9.0 is still broken on Windows (has been for 5 years). So we should not update it. I've replaced Jest with Vitest in all the repos where I need unit testing.